addons/revolutionaries/CfgVehicles_Vehicles.hpp

```diff
@@ -0,0 +1,155 @@
+//Revolutionaries (Tanoan)
+class C_Hatchback_01_F;
+class TACU_Revolutionaries_V_O_Hatchback01: C_Hatchback_01_F {
+    MACRO_CLEAR_VEHICLE_CARGO
+    faction = "TACU_Revolutionaries_O";
+	editorSubcategory = "TACU_Revolutionaries_EdSubCat_O_Cars_Tanoa";
```

Comment:
Incorrect formatting on editorSubcategory.
```suggestion
    editorSubcategory = "TACU_Revolutionaries_EdSubCat_O_Cars_Tanoa";
```